app/TuistApp/Sources/Extensions/SimulatorDeviceAndRuntime+Extras.swift

```diff
@@ -1,9 +1,18 @@
 import Foundation
 import TuistCore
 
-extension SimulatorDeviceAndRuntime: @retroactive Comparable {
-    public static func < (lhs: SimulatorDeviceAndRuntime, rhs: SimulatorDeviceAndRuntime) -> Bool {
-        if lhs.device.name == rhs.device.name { return lhs.runtime.name < rhs.runtime.name }
-        else { return lhs.device.name < rhs.device.name }
+#if swift(>=6)
+    extension SimulatorDeviceAndRuntime: @retroactive Comparable {
```

Comment:
Is this the only model that causes issues @sanghyeok-kim ?

Reply:
Yes, other cases work fine with Swift 5 without any issues.

Reply:
@pepicrft 

Oh, just found some additional issues in the test code.

Features introduced in [Swift Testing](https://www.swift.org/blog/announcing-swift-6/) are causing compilation errors in the Swift 5 environment:

- import Testing
- @Test
- @Suite
- #expect